@dprokop so when running this locally everything is green except the `@grafana/toolkit`.

I get the following error:


```
lerna ERR! yarn run build exited 1 in '@grafana-plugins/input-datasource'
lerna ERR! yarn run build stdout:
$ grafana-toolkit plugin:build
  Created: /Users/marcusandersson/development/go/src/github.com/grafana/grafana/plugins-bundled/internal/input-datasource/.prettierrc.js
  
  
  
  /Users/marcusandersson/development/go/src/github.com/grafana/grafana/plugins-bundled/internal/input-datasource/src/InputConfigEditor.tsx
    59:114  error  Replace `⏎··········datasource.·This·is·an·appropriate·place·to·enter·a·few·values.·Large·datasets·will·perform·better·in·other⏎·········` with `·datasource.·This·is·an·appropriate⏎··········place·to·enter·a·few·values.·Large·datasets·will·perform·better·in·other`  prettier/prettier
  
  /Users/marcusandersson/development/go/src/github.com/grafana/grafana/plugins-bundled/internal/input-datasource/src/InputDatasource.test.ts
    3:9  error  Replace `⏎··DataFrame,⏎··DataFrameDTO,⏎··DataSourceInstanceSettings,⏎··MutableDataFrame,⏎··PluginMeta,⏎··readCSV,⏎` with `·DataFrame,·DataFrameDTO,·DataSourceInstanceSettings,·MutableDataFrame,·PluginMeta,·readCSV·`  prettier/prettier
  
  /Users/marcusandersson/development/go/src/github.com/grafana/grafana/plugins-bundled/internal/input-datasource/src/testHelpers.ts
    4:59  error  Replace `⏎··options:·Partial<DataQueryRequest<TQuery>>⏎` with `options:·Partial<DataQueryRequest<TQuery>>`  prettier/prettier
  
  ✖ 3 problems (3 errors, 0 warnings)
    3 errors and 0 warnings potentially fixable with the `--fix` option.
  
  
  
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
- Preparing
✔ Preparing
- Linting
  Trace: Error: 3 linting errors found in 8 files
      at /Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/src/cli/tasks/plugin.build.ts:107:11
      at step (/Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/node_modules/tslib/tslib.js:136:27)
      at Object.next (/Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/node_modules/tslib/tslib.js:117:57)
      at fulfilled (/Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/node_modules/tslib/tslib.js:107:62)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at /Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/src/cli/utils/useSpinner.ts:13:15
      at step (/Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/node_modules/tslib/tslib.js:136:27)
      at Object.throw (/Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/node_modules/tslib/tslib.js:117:57)
      at rejected (/Users/marcusandersson/development/go/src/github.com/grafana/grafana/packages/grafana-toolkit/node_modules/tslib/tslib.js:108:69)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
✖ 3 linting errors found in 8 files
error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in '@grafana-plugins/input-datasource'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/packages/grafana-ui/.storybook/webpack.config.js
+++ b/packages/grafana-ui/.storybook/webpack.config.js
@@ -82,7 +82,7 @@ module.exports = ({ config, mode }) => {
       new TerserPlugin({
         cache: false,
         parallel: false,
-        sourceMap: true,
+        sourceMap: false,
       }),
       new OptimizeCSSAssetsPlugin({}),
     ],

